### PR TITLE
User experience update - start.cmd, docs

### PR DESCRIPTION
**Pull request by sushilraje**

### CODE REVIEW

File: docs/LOCAL_DEPLOYMENT.md [hunk 1]
```diff
@@ -1,52 +1,31 @@
...
-1) Run start.sh
-2) Run scripts under os folder. 
 
-#### Walk through for importing new Solution in IDE
+### Walk through for importing new Solution in IDE
 ##### Intellji
 This is our preferred editor for Java development.
 
-The java repository has a similar structure to the dotnet repo and contains the same script(s). The scripts, described above, are located under scripts/local/launch folder.  
+The java repository has a similar structure to the java repo and contains the same script(s). The scripts, described above, are located under scripts/local/launch folder.
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: undo java to dotnet. Or may be no need to call out that they are similar.

File: docs/LOCAL_DEPLOYMENT.md [hunk 2]
```diff
@@ -1,52 +1,31 @@
...
-2) create Azure resources 
-sh create_azure_resources.sh
- 
-After creating the required azure resources, using start or create-azure-resources.sh, one should execute the following scripts present under *os/{linux / win / osx}* to set the environment variables. 
-1) set-env-uri
-2) set-env
+#### Existing Users
+For users who have already created the required azure resources, please **set the environment variables** 
+1) Globally on your machine so as to be accessible by the IDE. OR
+2) In "Edit configurations" vizard of the Intellji IDE.
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: same comment as dotnet repo.

File: docs/LOCAL_DEPLOYMENT.md [hunk 3]
```diff
@@ -68,7 +47,19 @@ Environment variables can also be set in the Edit Configuration wizard under env
 The build.sbt file has been configured to run all the microservices parallelly. It will also start the microservices available only in .Net flavor (device-simulation, auth and ASA manager).
 
 *SBT tool has been observed to fork number of processes equivalent to the number of CPUs. Hence, on dual core machines all the microservices may not be started parallelly.*
- 
+
+
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: nit: extra lines

File: docs/LOCAL_DEPLOYMENT.md [hunk 4]
```diff
@@ -68,7 +47,19 @@ Environment variables can also be set in the Edit Configuration wizard under env
...
+#### Start Script
+The new repository contains a **start** script and few other scripts to bootstrap the new users with the required cloud resources. These scripts are used to create azure resources like Cosmos DB, IoTHub, Azure Stream Analytics etc. The start script is located in *scripts / local / launch* folder under root directory of the repository.
+
+#### Helpers scripts
+The script create-azure-resources.sh can be independently called to create resources in the cloud.
+##### Usage:
+1) create Azure resources   
+[create-azure-resources.(sh|cmd)](https://github.com/Azure/remote-monitoring-services-dotnet/blob/master/scripts/local/launch/helpers/create-azure-resources.sh)
+
+
```
Comment (ppathan) [inline docs/LOCAL_DEPLOYMENT.md]: nit: extra lines

File: scripts/local/launch/helpers/create-azure-resources.sh [hunk 5]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```
Comment (ppathan) [inline scripts/local/launch/helpers/create-azure-resources.sh]: You need to do a rename.

File: docs/LOCAL_DEPLOYMENT.md (hunk 1 above)
Comment (sushilraje) [reply docs/LOCAL_DEPLOYMENT.md]: ok

File: scripts/local/launch/helpers/create-azure-resources.sh (hunk 5 above)
Comment (sushilraje) [reply scripts/local/launch/helpers/create-azure-resources.sh]: I didn't get this

Comment (ppathan) [reply scripts/local/launch/helpers/create-azure-resources.sh]: create_azure_resources.sh was deleted and this file shows up as add. But it should be git mv which renames the file and only diffs would show if you made any changes.

Comment (sushilraje) [reply scripts/local/launch/helpers/create-azure-resources.sh]: I deleted and recreated the file with the same name.

**Pull request merged**